packages/gg_images/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+pkgname=gg_images
+pkgver=20.fe35153
+pkgrel=1
+pkgdesc='GG profile photo downloader'
+arch=('x86_64')
```

Comment:
Any reason `aarch64` is not included?

Reply:
I haven't tested it on the aarch64 architecture yet, but no reason it shouldn't work on that